lib/classes/ConfigSchemaHandler/index.js

```diff
@@ -136,6 +136,15 @@ class ConfigSchemaHandler {
             this.serverless.cli.log(`  ${message}`, 'Serverless', { color: 'orange' });
           }
         }
+        if (this.serverless.configurationInput.configValidationMode === undefined) {
+          this.serverless.cli.log(' ');
+          this.serverless._logDeprecation(
+            'CONFIG_VALIDATION_MODE_DEFAULT',
+            'Starting from next major Serverless will throw by default. Adapt to this' +
+              'behavior now by adding configValidationMode: error to service configuration'
```

Comment:
Let's go with `adding "configValidationMode: error"` - putting the setting between quotes will make it stand out from the message